lib/pages/feed/Picture.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/widgets.dart';
+
+class Picture extends StatelessWidget {
+  final String url;
+
+  Picture({@required this.url});
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
I believe that we can think about a more generic picture widget. We are using one here and there's another one in pixel_detail.dart

Reply:
Yes! i agree with you, maybe in the `pixel_detail.dart` we can use this new class... Can we do it in other PR?